streams/iq.m3u

```diff
@@ -1,20 +1,72 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="12ArabiaTV.iq",12 Arabia TV (1080p)
+https://livestream.almanasa.tv:8029/news/ch19/ch19_1080.m3u8
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="12ArabiaTV.iq",12 Arabia TV (1080p) [Geo-blocked]
https://livestream.almanasa.tv:8029/news/ch19/ch19_1080.m3u8
```

almanasa.tv doesn't work for me. If it works for you then please add `[Geo-blocked]` to every almanasa.tv link

Reply:
Yes it works, I will add [Geo-blocked] to it.